tests/components/HoursTable.test.tsx

```diff
@@ -138,9 +138,8 @@ describe("format", () => {
 describe("parseTime", () => {
   it("transforms Yext Time string to a JS localized time string", () => {
     const actual = parseTime("09:00");
-    const expected = "9:00 AM";
 
-    expect(actual).toBe(expected);
+    expect(actual).toMatch(/^9:00[\s\p{u+202f}]AM$/);
```

Comment:
Why did we need this regex change? It seems less readable.

Reply:
The test was broken in Node 18. In Node 18, the `\s` is not used between '9:00' and 'AM'.

Reply:
I see, maybe we can slightly adjust this expect case to show there's only two possibilities? I was thinking `expect(["9:00AM", "<whatever the node 18 value is>"]).toContain(actual)`. I think that'll make it easier to investigate later